pkgs/development/python-modules/hebg/default.nix

```diff
@@ -0,0 +1,59 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+
+  # Runtime dependencies
+  networkx,
+  matplotlib,
+  numpy,
+  tqdm,
+  scipy,
+
+  # Build, dev and test dependencies
+  git,
+  setuptools-scm,
+  pytestCheckHook,
+  pytest-check,
+  pytest-mock,
+  sphinx,
+}:
+
+buildPythonPackage rec {
+  pname = "hebg";
+  version = "0.2.4";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-11bz+FbnaEVLiXT1eujMw8lvABlzVOeROOsdVgsyfjQ=";
+  };
+
+  build-system = [
+    git
+    setuptools-scm
+  ];
+
+  propagatedBuildInputs = [
+    networkx
+    matplotlib
+    numpy
+    tqdm
+    scipy
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    pytest-check
+    pytest-mock
+  ];
+
+  meta = {
+    description = "Hierarchial Explainations of Behavior as Graph";
+    homepage = "https://github.com/IRLL/HEB_graphs";
+    downloadPage = "https://github.com/IRLL/HEB_graphs";
```

Comment:
```suggestion
```
It is the same as homepage and does not need to be set.

Reply:
Removed